### PR TITLE
* Update assets icons.

### PR DESCRIPTION
**Pull request by megax**
Remove old files.

### CODE REVIEW

**Comment by megax** [comment]
@khurshid-alam updated. Please testing :)

**Comment by lots0logs** [comment]
@megax Does this PR have a corresponding github issue? We want to try and keep all discussion of PRs on a corresponding github issue.  Anyway, just a quick note from looking over these changes: The PR is dirty--It includes some changes that arent part of the stated purpose of the PR.

**Comment by megax** [comment]
@lots0logs https://github.com/numixproject/numix-gtk-theme/pull/526#issuecomment-242443439 @khurshid-alam request :)

**Comment by khurshid-alam** [comment]
@megax This PR is still dirty. As per my view only 37 files (png-> svg) should get modified...nothing else. And the issue would be: #508 . By mistake I closed it.

**Comment by megax** [comment]
There were some old file folder, and i deleted png files. I updated gtk.gresource.xml and sass files. Stay png also?

**Comment by khurshid-alam** [comment]
Ok. I see. You removed pngs altogether and overwrite existing svgs. It should be ok though. Let me check.

**Comment by khurshid-alam** [comment]
@megax Bug (nautilus) no checkbox ?

nautilus-no-checkbox

**Comment by megax** [comment]
Svg bug? Then stay png.

File: gtk-3.0/scss/widgets/_toggle.scss [hunk 1]
```diff
@@ -7,41 +7,41 @@ $suffix: if($variant == "dark", "-dark", "");
 @mixin toggle($type) {
     background-image: none;
 
-    -gtk-icon-source: url("../assets/#{$type}-unchecked#{$suffix}.png");
+    -gtk-icon-source: url("../assets/#{$type}-unchecked#{$suffix}.svg");
 
-    &:insensitive { -gtk-icon-source: url("../assets/#{$type}-unchecked-insensitive#{$suffix}.png"); }
+    &:insensitive { -gtk-icon-source: url("../assets/#{$type}-unchecked-insensitive#{$suffix}.svg"); }
```
Comment (khurshid-alam) [inline gtk-3.0/scss/widgets/_toggle.scss]: @megax  `-gtk-icon-squrce`? Please fix that

**Comment by khurshid-alam** [comment]
Svg bug. The reason is new `checkbox-unchecked.svg` doesn't have any border and background color overlaps with theme_base_color. That is why it appears invisible. `checkbox-unchecked.png` that we are using now was generated from previous assets.

For the time being we will be keep using pngs until these problems get fixed.

**Comment by megax** [comment]
Okay, I create new PR only svg files update?

**Pull request closed without merging**